src/Core/UbuntuDriversBackend.vala

```diff
@@ -47,15 +47,10 @@ public class AppCenterCore.UbuntuDriversBackend : Backend, Object {
         }
 
         working = true;
+
         cached_packages = new Gee.TreeSet<Package> ();
-        string? command_output;
-        var result = yield get_drivers_output (cancellable, out command_output);
-        if (!result || command_output == null || cancellable.is_cancelled ()) {
-            working = false;
-            return cached_packages;
-        }
+        var tokens = AppCenter.App.settings.get_strv ("cached-drivers");
```

Comment:
The `ubuntu-drivers` command takes a long time to run, so just lookup the driver package names we used last time from a GSettings cache.